data/human/human missions.txt

```diff
@@ -6849,6 +6849,133 @@ mission "FW Refugees to Humanika"
+			choice
+				`	(Follow the map.)`
+					accept
+				`	(Throw the map away, it's probably fake.)`
+					decline
+	on complete
+		conversation
+			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shovelling the hard soil you're met with a thunk as you strike metal.`
+			branch hai
+				has "First Contact: Hai: offered"
+			`	Opening the cache reveals a large store of credits along with the alien weapons the map's owner wrote about. You marvel at their strange design and wonder at their power.`
+			choice
+				`	(Loot the treasure.)`
+					goto end
+			label hai
+			`	Opening the cache reveals a large store of credits along with two Hai Ion Cannons. These must be the alien weapons the map's owner wrote about.`
+			choice
+				`	(Loot the treasure.)`
+					goto end
+			label end
+			action
+				payment 386000
+				outfit "Ion Cannon" 2
+			`	You grab the <payment> credits, and with the assistance of some machinery you bring the weapons back to your ship.`
+
+mission "Derelict Family Investment 1"
+	name "Lost Family Investment"
+	description "Return the credits you found in a derelict ship to the family it belongs to."
+	name "Colin Sawyer's Investment"
+	landing
+	to offer
+		has "derelict family investment"
+	destination "Sundive"
+	on offer
+		conversation
+			`Upon landing on <origin> you take out the text log you recovered alongside the case of credits from the derelict ship. The log reveals the credits were the result of a risky investment in an attempt to buy a better life for the captain of the ship, Colin Sawyer, and his family. You assume that the captain never made it home, and looking further into the personal log you find that the family used to reside on <destination>.`
+			`	Perhaps the family still live there?`
+			choice
+				`	(Attempt to return the credits.)`
+					accept
+				`	(Keep the credits to yourself.)`
+			action
+				payment 325000
+			`	You decide to keep the money. After all, it's not every day that you find a score worth <payment>.`
+				decline
+	on complete
+		conversation
+			`	Upon landing on <planet> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`
+			choice
+				`	"Do you know a Colin Sawyer?"`
+				`	"Do the Sawyers live here?"`
+			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyer's new place of residence and bids you farewell.`
```

Comment:
This should be a part of the on offer conversation of the next mission. That way you can actually tell the player what the destination is with a text replacement.